cmd/create/runner.go

```diff
@@ -82,6 +82,7 @@ func fetchAndDiff(dir string) (string, error) {
 			"--name-status",   // only show filename and the type of change (A=added, etc.)
 			"origin/master",   // diff against the latest master
 			"--diff-filter=A", // only show added files
+			"--no-renames",    // disable rename detection so we always find new releases
```

Comment:
If a `releases` branch being tested is outdated and a new release is merged to `master` which is similar, `git` detects the difference as a rename instead of an added file. This flag avoids that situation. Tested against https://github.com/giantswarm/releases/pull/394